sql/mob_skills.sql

```diff
@@ -438,7 +438,7 @@ INSERT INTO `mob_skills` VALUES (630,374,'Kick_Out',4,10.0,2000,1500,4,0,0,0);
 INSERT INTO `mob_skills` VALUES (631,375,'Shock_Wave',4,7.0,2000,1500,4,0,0,2);
 INSERT INTO `mob_skills` VALUES (632,376,'Flame_Armor',0,7.0,2000,1500,1,0,0,0);
 INSERT INTO `mob_skills` VALUES (633,354,'Howl',1,20.0,2000,1500,1,0,0,0);
--- INSERT INTO `mob_skills` VALUES (634,378,'Meteor',0,7.0,2000,1500,4,0,0,0);
+-- INSERT INTO `mob_skills` VALUES (634,2256,'Meteor',0,7.0,2000,1500,4,0,0,0);
```

Comment:
explains why when I checked that it looked like humping the ground, must be specific to that one NM I presume. Amend incoming.